packages/suite/src/views/settings/SettingsCoins/SettingsCoins.tsx

```diff
@@ -1,26 +1,49 @@
 import styled from 'styled-components';
-
+import { AnimatePresence, motion } from 'framer-motion';
 import { hasBitcoinOnlyFirmware, isBitcoinOnlyDevice } from '@trezor/device-utils';
-import { selectSupportedNetworks } from '@suite-common/wallet-core';
+import { selectSupportedNetworks, startDiscoveryThunk } from '@suite-common/wallet-core';
+import { Button, motionEasing } from '@trezor/components';
 
 import { DeviceBanner, SettingsLayout, SettingsSection } from 'src/components/settings';
 import { CoinGroup, SectionItem, TooltipSymbol, Translation } from 'src/components/suite';
 import { useEnabledNetworks } from 'src/hooks/settings/useEnabledNetworks';
 import { useAnchor } from 'src/hooks/suite/useAnchor';
 import { SettingsAnchor } from 'src/constants/suite/anchors';
-import { useDevice, useSelector } from 'src/hooks/suite';
+import { useDevice, useDiscoveryButtonVisible, useDispatch, useSelector } from 'src/hooks/suite';
 
 import { FirmwareTypeSuggestion } from './FirmwareTypeSuggestion';
 
-const StyledSettingsLayout = styled(SettingsLayout)`
-    & > * + * {
-        margin-top: 16px;
-    }
+const StyledSettingsSection = styled(SettingsSection)`
+    overflow: hidden;
 `;
 
+const discoveryButtonAnimationConfig = {
```

Comment:
Currently this causes a jump because of some SectionItem styles but should be gone once rebased!